src/swell/utilities/slurm.py

```diff
@@ -6,7 +6,9 @@
 
 # --------------------------------------------------------------------------------------------------
 
-import os
+import importlib.resources
```

Comment:
To expose some additional `importlib` functionality:

```suggestion
import importlib
```